src/core/agent/records/credentialStorage.test.ts

```diff
@@ -15,7 +15,7 @@ const storageService = jest.mocked({
   getAll: jest.fn(),
 });
 
-const credentialStorage = new CredentialStorage(storageService as any);
+const credentialStorage = new CredentialStorage(storageService as never);
```

Comment:
`new CredentialStorage(storageService as StorageService<CredentialMetadataRecord>);` should work here and other places